lxd/db/cluster/certificate_projects.go

```diff
@@ -7,6 +7,7 @@ package cluster
 //go:generate -command mapper lxd-generate db mapper -t certificate_projects.mapper.go
 //go:generate mapper reset -i -b "//go:build linux && cgo && !agent"
 //
+//go:generate mapper stmt -e certificate_project objects
```

Comment:
What does "Add objects generation for..." mean and why is this only being applied to instance_profiles and certificate_projects

Reply:
So the primary change here is that the `objects-by` directives now parse the variable declaration that's produced from the `objects` directive (or an equivalent variable declaration created manually). These two entities didn't have an `objects` directive, so their `objects-by` directives would have failed.